internal/backend/ioutils/intcomp_test.go

```diff
@@ -0,0 +1,38 @@
+package ioutils
+
+import (
+	"bytes"
+	"testing"
+)
+
+func FuzzIntcomp32(f *testing.F) {
```

Comment:
Fuzzing becomes more efficient with some seeded corpus. I ran it for a few minutes and built one. Should I included it? It also becomes regression test in the sense that when `FuzzIntcomp32` is run as a unit test not a fuzzing test then checks all the cases defined in the corpus.

Reply:
And as a general note, Go integrated fuzzing seems very powerful, I think we should aim to include it with more in serialization tests in general.

Reply:
oh yes feel free to include it !

Reply:
Added test corpus